backend/routes/restaurant.js

```diff
@@ -25,6 +26,39 @@ router.get('/', (req, res) => {
   );
 });
 
+// Get Specific Restaurant opening hours
+router.get('/openhours', (req, res) => {
+  const input = req.query;
+  if (!input.restaurantID) {
+    res.status(400).json({ error: '/restaurant/openhours GET endpoint needs a restaurantID query param' });
+    return;
+  }
+
+  connection.query(
+    `SELECT restaurant_db.HOURS.DayOfWeek, restaurant_db.HOURS.OpenTime,restaurant_db.HOURS.CloseTime 
+    from restaurant_db.HOURS 
+    left join restaurant_db.RESTAURANT 
+    ON restaurant_db.HOURS.RestaurantID = restaurant_db.RESTAURANT.ID
+    Where restaurant_db.RESTAURANT.ID = ${req.query.restaurantID}`,
```

Comment:
Should this be ${input.restaurantID}? Functionally the same but might be better for readability

Reply:
Good idea. Updated to use ${input.restaurantID}